cclib/parser/fchkparser.py

```diff
@@ -0,0 +1,229 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2020, the cclib development team
+#
+# This file is part of cclib (http://cclib.github.io) and is distributed under
+# the terms of the BSD 3-Clause License.
+
+"""Parser for Formatted Checkpoint files"""
```

Comment:
Could you add a bit more context here? What kind of data is typically stored, and what basic programs are supported?

Reply:
Are you hinting at something specific like these files store molecule and wavefunction information (e.g. `moenergies`, `mocoeffs`)?

Reply:
I think so...and that these files are produced by Q-Chem and Gaussian (any others?).